nestoria-project/src/app/modules/property-search-page/property-search.component.css

```diff
@@ -0,0 +1,41 @@
+.nav{
```

Comment:
Update your CSS - to SCSS, this will be easier for you to develop CSS code, after updating upgrade your CSS to SCSS with best practices

Reply:
updated